subsys/bluetooth/controller/ll_sw/ull_slave.c

```diff
@@ -213,6 +213,20 @@ void ull_slave_setup(memq_link_t *link, struct node_rx_hdr *rx,
 		}
 	}
 
+#if defined(CONFIG_BT_CTLR_ADV_EXT)
+	if (adv->is_created & BIT(1)) {
+		ll_rx_put(link, rx);
+
+		/* use reserved link and node_rx to prepare
```

Comment:
Note that elsewhere comment style is with each comment-line starting with '*', and the '*/ at a separate line